Description: fuser -M always on
 Last psmisc update had a typo which means -M is always on
Bug-Debian: http://bugs.debian.org/740275
Author: Brad Jorsch <anomie@users.sourceforge.net>
Last-Update: 2014-02-27
--- a/src/fuser.c
+++ b/src/fuser.c
@@ -1174,7 +1174,7 @@
 		usage(_("No process specification given"));
 
 	/* Check if -M flag was used and if so check mounts */
-	if (opts * OPT_ISMOUNTPOINT) {
+	if (opts & OPT_ISMOUNTPOINT) {
 	    check_mountpoints(&mounts, &names_head, &names_tail);
 	}
 
